recipes/graaf/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=2"
+
+
+class GraafConan(ConanFile):
+    name = "graaf"
+    description = (
+        "Graaf: A Lightweight, Header-Only C++20 Graph Library."
+    )
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bobluppes/graaf"
+    topics = ("graph", "header-only")
+    package_type = "header-library"
+    short_paths = True
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def package_id(self):
+        self.info.clear()
+
+    def validate(self):
+        check_min_cppstd(self, 20)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def package(self):
+        copy(self, "LICENSE.md", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "*",
+                  src=os.path.join(self.source_folder, "include"),
+                  dst=os.path.join(self.package_folder, "include"))
+
+    def package_info(self):
+        self.cpp_info.bindirs = []
+        self.cpp_info.libdirs = []
+
+        self.cpp_info.set_property("cmake_file_name", "Graaf")
```

Comment:
In https://bobluppes.github.io/graaf/docs/quickstart/installation#cmake-fetchcontent, the fetch content declaration shows:

> ```
>include(FetchContent)
>FetchContent_Declare(
>        graaflib
>        GIT_REPOSITORY https://github.com/bobluppes/graaf.git
>        GIT_TAG main
>)
>FetchContent_MakeAvailable(graaflib)
>```
>Now you can link your target against Graaf::Graaf:
>```
>target_link_libraries(${PROJECT_NAME} PRIVATE Graaf::Graaf)
>```

So it seems they expect the find name to be `graaflib` instead of `Graaf`

```suggestion
        self.cpp_info.set_property("cmake_file_name", "graaflib")
```